# HG changeset patch
# User A.R. Burgers <arburgers@gmail.com>
# Date 1524066410 25200
#      Wed Apr 18 08:46:50 2018 -0700
# Branch stable
# Node ID cc76e2ce0a99090a0711f7a4de3bc5093117e1b4
# Parent  692fbde19871c2cc815d7cc7e0581770fede143c
nl_NL.ts: Fix single typo in Dutch translation (bug #53687).

* nl_NL.ts: Fix single typo in Dutch translation (bug #53687).

diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -539,17 +539,17 @@ Wil je die toch verwijderen?</translatio
     <message>
         <location line="+1"/>
         <source>function</source>
         <translation>functie</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>global</source>
-        <translation>gobaal</translation>
+        <translation>globaal</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>hidden</source>
         <translation>verborgen</translation>
     </message>
     <message>
         <location line="+1"/>
